scripts/docker-for-test.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 if [ -n "$(which docker)" ]; then
-    DOCKER_IMAGES=("mysql:5.7" "postgres:9.6")
+    DOCKER_IMAGES=("mysql:5.7" "postgres:9.6" "quay.io/skuid/docker-aws-redshift")
```

Comment:
these docker images are for connection breaking etc tests, not used for general travis CI integration tests.